nearcore/src/runtime/mod.rs

```diff
@@ -93,9 +93,6 @@ impl NightshadeRuntime {
         if config.config.store.state_snapshot_enabled {
             state_snapshot_type = StateSnapshotType::EveryEpoch;
         }
-        if let Some(n) = config.client_config.state_snapshot_every_n_blocks {
```

Comment:
We can remove `state_snapshot_every_n_blocks` from client_config as well?

Reply:
Yes, will add this commit in a moment.

Reply:
Done